packages/remix-testing/create-remix-stub.tsx

```diff
@@ -8,23 +8,30 @@ import type {
   RouteModules,
 } from "@remix-run/react";
 import { RemixEntry } from "@remix-run/react";
+import { json } from "@remix-run/server-runtime";
+
 import type {
+  Action,
   AgnosticDataRouteObject,
   AgnosticIndexRouteObject,
   AgnosticNonIndexRouteObject,
+  AgnosticRouteObject,
   AgnosticRouteMatch,
   InitialEntry,
   Location,
   MemoryHistory,
   StaticHandler,
-  Update,
-} from "@remix-run/server-runtime";
+} from "@remix-run/router";
 import {
   createMemoryHistory,
-  json,
   matchRoutes,
   unstable_createStaticHandler as createStaticHandler,
-} from "@remix-run/server-runtime";
+} from "@remix-run/router";
+
+type Update = {
+  action: Action;
+  location: Location;
+};
```

Comment:
This isn't exported through the router since history is an implementation detail now, but it could be if needed.  Just defined it here for now though